src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -352,7 +352,7 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
   @Override
   public void run() {
     FrameworkInfo.Builder frameworkInfo = FrameworkInfo.newBuilder()
-        .setName("HDFS " + conf.getClusterName())
+        .setName(conf.getClusterName())
```

Comment:
Talked to @abhay-agarwal, we cannot have spaces in the framework name for Mesos DNS to work. HDFS as the default name should be fine. Let's change this to conf.getFrameworkName() to be more precise than conf.getClusterName() and set the default value to HDFS.